pom.xml

```diff
@@ -211,7 +501,7 @@
     </plugins>
   </build>
 
-  <dependencyManagement>
+  <!--<dependencyManagement>
```

Comment:
Please remove or uncomment.

Reply:
The `dependencyManagement` has been removed.